src/main/java/com/google/sps/data/ChromeOSDevice.java

```diff
@@ -15,6 +15,7 @@
   private String annotatedUser;
   private final String deviceId;
   private final String serialNumber;
+  private final String status;
```

Comment:
is it used?

Reply:
Yes.  When we use Gson to deserialize the chromebook data sent by the Directory API, it creates the object by examining each of its fields and searching the json for a field with a matching name.  So in this case, adding a `status` field to the ChromeOSDevice class means when we deserialize the incoming json, Gson will look for a field called `status` in the json and assign that value to the ChromeOSDevice's status field.  It does not actually use the class's constructor in any way, which is why we have decided to just manually assign it a value of 'provisioned' in the constructor-- this value would only ever be used in tests where we manually create ChromeOSDevices.  After being assigned, the status field is eventually passed to the client when the ChromeOSDevice is serialized, and the client then displays the status.